<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_main</class>
 <widget class="QMainWindow" name="frm_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>whatever</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_charaktere">
     <attribute name="title">
      <string>Roster</string>
     </attribute>
     <widget class="QTableView" name="tbl_charaktere">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>1131</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_update">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>580</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Roster</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_update_sims">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>580</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Sims</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_items">
     <attribute name="title">
      <string>Sim Results</string>
     </attribute>
     <widget class="QComboBox" name="ddown_item_filter_dungeon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="list_items">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>221</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_item_id">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_item_id">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_item_name">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_item_name">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_item_source_dungeon">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_item_slot">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Slot</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_item_slot">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_item_source_dungeon">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Dungeon</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_item_source_encounter">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Encounter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_item_source_encounter">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="ddown_item_filter_encounter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_item_search">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tbl_sim_results">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>541</width>
        <height>541</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_item_filter_dungeon_submit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Dungeon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_item_filter_encounter_submit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Encounter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_item_search">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Suche</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_advanced">
     <attribute name="title">
      <string>Advanced</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_log">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextEdit" name="text_log_viewer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1131</width>
        <height>601</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_test">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>700</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
